x/ref/runtime/internal/flow/manager/manager.go

```diff
@@ -569,7 +571,48 @@ func (m *manager) lnAcceptLoop(ctx *context.T, ln flow.Listener, local naming.En
 				m.acceptChannelTimeout,
 				fh)
 			if err != nil {
-				ctx.Errorf("failed to accept flow.Conn on localEP %v failed: %v", local, err)
+				// We don't want probing from load balancers or Prometheus to cause
+				// the error log to be noisy so we skip logging an err in the following
+				// cases:
+				//
+				// (1) 'message of type 127 and size 0 failed decoding at field 0: <nil>'
+				//
+				// This can be trigger using "echo -e '\xff\xff\xff' | nc ...".
+				//
+				// (2) 'error reading from unknown: EOF'
+				//
+				// This can be trigger using nc to connect to a tcp Vanadium endpoint
+				// and then typing Ctrl-C after receiving the first message. Another way
+				// is using 'nc ... & p=$! && sleep 1 && kill -s SIGINT ${p}'
+				//
+				// (3) 'read: connection reset by peer'.
+				//
+				// This can be trigger using 'nmap -sT' to a tcp Vanadium endpoint.
+				skip := false
+				if verror.ErrorID(err) == conn.ErrRecv.ID {
+					verr := err.(verror.E)
+					if verr.ParamList[3] == io.EOF {
+						skip = true
+					}
+					switch p := verr.ParamList[3].(type) {
+					case *net.OpError:
+						if p.Err.(*os.SyscallError).Err == syscall.ECONNRESET {
```

Comment:
I ran into panics with the mounttables in the analysis pipeline clusters that were caused by this type assertion.  Some of the time p.Err seems to be an *errors.errorString and not *os.SyscallError:

> panic: interface conversion: error is *errors.errorString, not *os.SyscallError
> 
> goroutine 10384 [running]:
> grail.com/vendor/v.io/x/ref/runtime/internal/flow/manager.(*manager).lnAcceptLoop.func2(0xc4204fa640, 0xc02e00, 0xc422e8f500, 0xb712f4, 0x3, 0xc4201bf790, 0x10, 0x2f0cfb92d408958b, 0x632aa3a83473d8f1, 0x0, ...)
> 	/home/rck/trees/grail/go/src/grail.com/vendor/v.io/x/ref/runtime/internal/flow/manager/manager.go:599 +0xc3d
> created by grail.com/vendor/v.io/x/ref/runtime/internal/flow/manager.(*manager).lnAcceptLoop
> 	/home/rck/trees/grail/go/src/grail.com/vendor/v.io/x/ref/runtime/internal/flow/manager/manager.go:562 +0x46f

Can you modify this to be:

```
serr, ok := p.Err.(*os.SyscallError)
if ok && serr.Err == syscall.ECONNRESET {
```
instead?  Or, if you could merge this change onto master then I can add a pull request with the fix.  

Thanks, I hope you're doing well!

Reply:
Thanks for pointing that out. I'll update the PR to do that later today. :-)